proto/agent.proto

```diff
@@ -294,14 +294,26 @@ message IOStat {
 	float writeBytesRate = 4;
 }
 
+enum ConnectionType {
+	tcp = 0;
+	udp = 1;
+}
+
+enum ConnectionFamily {
+	v4 = 0;
+	v6 = 1;
+}
+
 message Connection {
 	int32 pid = 1;
-	int32 fd = 2;
+	// int32 fd = 2;         // Unused for now -- may be deprecated
 	int32 family = 3;
```

Comment:
change the type to ConnectionFamily/ConnectionType? (might need to be a new field, but it seems cleaner IMO).